ikasaneip/component/endpoint/email-endpoint/src/main/java/org/ikasan/component/endpoint/email/producer/EmailProducer.java

```diff
@@ -1,207 +0,0 @@
-package org.ikasan.component.endpoint.email.producer;
-
-import org.apache.log4j.Logger;
-import org.ikasan.spec.component.endpoint.EndpointException;
-import org.ikasan.spec.component.endpoint.Producer;
-import org.ikasan.spec.configuration.ConfiguredResource;
-
-import javax.activation.DataHandler;
-import javax.activation.DataSource;
-import javax.mail.*;
-import javax.mail.internet.*;
-import javax.mail.util.ByteArrayDataSource;
-import java.util.List;
-import java.util.Properties;
-/**
- * Created by xualys on 16/09/2015.
- */
-public class EmailProducer implements Producer<EmailPayload>, ConfiguredResource<EmailProducerConfiguration> {
```

Comment:
Make this a ManagedResource as it interacts with an external service, so start/stop are relevant for this component